mybatis-spring-boot-autoconfigure/src/main/java/org/mybatis/spring/boot/autoconfigure/MybatisAutoConfiguration.java

```diff
@@ -206,7 +208,9 @@ public void setResourceLoader(ResourceLoader resourceLoader) {
 
     @PostConstruct
     public void afterPropertiesSet() {
-      log.debug(String.format("No %s found.", MapperFactoryBean.class.getName()));
+      if (log.isDebugEnabled()) {
+        log.debug(String.format("No %s found.", MapperFactoryBean.class.getName()));
+      }
```

Comment:
I would agree if the logging hit was needed and generally I see string format in a logger as bad practice.  After looking at the class a bit, I see an obvious flaw in logging being used.  It's using apache commons logging (legacy) but also has slf4j available.  If we switched this to use slf4j instead, the cost can be incurred inside the debug statement rather than external to it.  However, that is beyond this pull request so for the time being this is ok.  I'm merging it now.  Thanks.